src/taxonomy/taxonomy-detail/TaxonomyDetailPage.jsx

```diff
@@ -0,0 +1,128 @@
+// ts-check
+import React, { useState } from 'react';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Breadcrumb,
+  Container,
+  Layout,
+} from '@edx/paragon';
+import { Link, useParams } from 'react-router-dom';
+
+import ConnectionErrorAlert from '../../generic/ConnectionErrorAlert';
+import Loading from '../../generic/Loading';
+import SubHeader from '../../generic/sub-header/SubHeader';
+import taxonomyMessages from '../messages';
+import TaxonomyDetailMenu from './TaxonomyDetailMenu';
+import TaxonomyDetailSideCard from './TaxonomyDetailSideCard';
+import { TagListTable } from '../tag-list';
+import ExportModal from '../export-modal';
+import { useTaxonomyDetailDataResponse, useTaxonomyDetailDataStatus } from './data/apiHooks';
+
+const TaxonomyDetailPage = () => {
+  const intl = useIntl();
+  const { taxonomyId } = useParams();
+
+  const useTaxonomyDetailData = () => {
+    const { isError, isFetched } = useTaxonomyDetailDataStatus(taxonomyId);
+    const taxonomy = useTaxonomyDetailDataResponse(taxonomyId);
+    return { isError, isFetched, taxonomy };
+  };
+
+  const { isError, isFetched, taxonomy } = useTaxonomyDetailData(taxonomyId);
```

Comment:
This internal hook doesn't seem to be reused so I'm not sure why it's needed. 
```suggestion
  const { isError, isFetched } = useTaxonomyDetailDataStatus(taxonomyId);
  const taxonomy = useTaxonomyDetailDataResponse(taxonomyId);
```

Reply:
Addressed with 0e19e99